docker/install.sh

```diff
@@ -23,31 +23,61 @@ set -ex
 # ROS1 installation
 ###
 # Source the autoware installation
-source /opt/autoware.ai/ros/install/setup.bash
+
+if [[ ! -z "$ROS1_PACKAGES$ROS2_PACKAGES" ]]; then
```

Comment:
Is having no space between these two variables ok?
I'm thinking its probably fine if checking for not empty but want to confirm.

Reply:
Yeah, it's just concatenating them and checking if the result is empty, I'm using this conditional a lot to see if the incremental build option was selected without need for a separate flag.